README.md

```diff
@@ -1,4 +1,174 @@
+
+    Units used by all the bands in the evalscript. If `None`, `units` evalscript parameter isn't set and default units for each band are used. See [docs](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#input-object-properties).
+
+* **`mosaicking`**: *str, optional*. Default: `ORBIT`
+
+    Works with multi-temporal data by default. See [possible values](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#mosaicking).
+
+* **`bands_dimension_name`**: *str, optional*. Default: `bands`
+
+    Name of the default dimension of type `bands` of the datacube, as set in `load_collection` and referred to in the process graph.
+
+* **`temporal_dimension_name`**: *str, optional*. Default: `t`
+
+    Name of the default dimension of type `temporal` of the datacube, as set in `load_collection` and referred to in the process graph.
+
+* **`datacube_definition_directory`**: *str, optional*. Default: `javascript_datacube`
+
+    Relative path to the directory with the javascript implemetations of the processes.
+
+* **`output_dimensions`**: *list of dicts or None, optional*. Default: `None`
+
+    Information about the dimensions in the output datacube. This can be set if the value is known beforehand. Each element contains:
+    - `name`: name of the dimension,
+    - `size`: size (length) of the dimension,
+    - `original_temporal`, optional: boolean, should be `True` if this is the temporal dimension generated in the initial `load_collection` node.
+
+###### Methods
+
+* **`write`** ():
+
+    Returns the evalscript as a string.
+
+* **`determine_output_dimensions`** ()
+
+    Calculates the greatest possible dimensions of the output datacube, returning a list of dicts. Each element contains:
+    - `name`: name of the dimension,
+    - `size`: size (length) of the dimension,
+    - `original_temporal`, optional: boolean, `True` if this is the temporal dimension generated in the initial `load_collection` node.
+
+* **`set_output_dimensions`** (output_dimensions):
+
+    Setter for output dimensions. `output_dimensions` is a list of dicts. Each element contains:
+    - `name`: name of the dimension,
+    - `size`: size (length) of the dimension,
+    - `original_temporal`, optional: boolean, should be `True` if this is the temporal dimension generated in the initial `load_collection` node.
+
+* **`get_decoding_function`** ():
+
+    Returns a `decode_data` function. The data returned by the evalscript is encoded to contain the information about the datacube dimensions and has to be decoded to obtain the actual data in a ndarray format.
+    `decode_data` has the following parameters:
+    - `data`: the result of processing of the associated evalscript, it should be a three-dimensional array.
+    `decode_data` returns a multidimensional Python list.
+
+
+## Workflow
+
+1. Construct the process graph
+
+Load a file with `json.load` or generate a process graph using [openEO Python client](https://github.com/Open-EO/openeo-python-client).
```

Comment:
Nitpicking

2 added to the beginning of the line would indent it so that it wouldn't stick out so much. (similarly as in [EDC Browser's readme](https://git.sinergise.com/team-6/edc-browser/-/blob/master/README.md#L54))

I think it can be done the same for code (text wrapped in  \`\`\`), but not sure if every line should have those additional spaces at the beginning.

FFTR.